packages/openneuro-app/src/scripts/datalad/dashboard/datasets/dataset-query.jsx

```diff
@@ -161,6 +162,7 @@ const datasetQueryDisplay = isPublic => ({
       refetch={refetch}
       queryVariables={variables}
       publicDashboard={isPublic}
+      isMobile={useMedia('(max-width: 700px) ')}
```

Comment:
The mobile breakpoint is 765 elsewhere so this creates an odd looking intermediate state.